syntaxes/src/template/index.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {GrammarDefinition} from '../types';
+
+import {bindingKey} from './repository/binding-key';
+import {eventBinding} from './repository/event-binding';
+import {interpolation} from './repository/interpolation';
+import {propertyBinding} from './repository/property-binding';
+import {templateBinding} from './repository/template-binding';
+import {twoWayBinding} from './repository/two-way-binding';
+
+export const template: GrammarDefinition = {
+  scopeName: 'template.ng',
+  injectionSelector: 'L:text.html -comment',
+  patterns: [
+    {include: '#interpolation'},
+    {include: '#propertyBinding'},
+    {include: '#eventBinding'},
+    {include: '#twoWayBinding'},
+    {include: '#templateBinding'},
+  ],
+  repository: {
+    interpolation,
+    propertyBinding,
+    eventBinding,
+    twoWayBinding,
+    templateBinding,
+    bindingKey,
```

Comment:
I would prefer to place all of these definitions in this file, since they are exclusive to the template grammar and very similar, making locality easier to browser. What do you think?

Reply:
I personally prefer shorter and separated files because I can open them in multiple panes and look at them side-by-side vs having to scroll all over the place. Especially if/when the grammar becomes more complex, I think this will be more sustainable in the long run vs having a single file that's thousands of lines long, but I suppose that's speculative at this point. It looks like there's no guidance in the Google style guide about file lengths (although [Angular's style guide](https://angular.io/guide/styleguide#rule-of-one) recommends a 400-line limit).

I'll let you consider that and make a final decision one way or the other. :) It's not going to break my heart to consolidate, but that would be my argument for leaving it as-is.

Reply:
I think we should consolidate in one file because

- symmetry with the outputted JSON file
- the template grammar is a small superset of the HTML grammar, and probably won't be expanded much more than it is now (because we won't rewrite the HTML grammar)

Reply:
@ayazhafiz Sorry for taking so long to get to this, long day at work. Updated with a10718c!